plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/CitrixResourceBase.java

```diff
@@ -2592,9 +2592,10 @@ public VDI getIsoVDIByURL(final Connection conn, final String vmName, final Stri
         String mountpoint = null;
         if (isoURL.startsWith("xs-tools")) {
```

Comment:
This ISO URL remained the same?

Reply:
Yes it did. I had an idea to change the iso name (url here refers to _name_) to something general - like `xenserver-tools` and manage the same thing in `actualIsoTemplate`, but _xs-tools.iso_ is used in some many places (specially in some test cases) which I ended up not changing this. Potentially it only needs a wiki/user guide doc only.